File: posgret/conexion.rb
require 'pg'

begin
  # Crear la conexión a PostgreSQL
  conn = PG.connect(
    host: "localhost",
    dbname: "mi_basedatos",
    user: "postgres",
    password: "1234" # reemplaza con tu contraseña real
  )

  puts "✅ Conexión exitosa a la base de datos conex3066478"

  # Hacer una consulta
  res = conn.exec("SELECT * FROM productos")

  # Mostrar los resultados
  res.each do |fila|
    puts fila
  end

rescue PG::Error => e
  puts "❌ Error al conectar o consultar: #{e.message}"
ensure
  conn.close if conn
end

File: posgret/conexion.js
const { Client } = require('pg');

const client = new Client({
  host: 'localhost',
  user: 'postgres',
  password: '1234',  // reemplaza con tu contraseña real
  database: 'mi_basedatos',
  port: 5432, // puerto por defecto de PostgreSQL
});

client.connect(err => {
  if (err) {
    console.error('❌ Error al conectar a PostgreSQL:', err.stack);
  } else {
    console.log('✅ Conexión exitosa a PostgreSQL');
  }
});

